eng/Dependencies.props

```diff
@@ -8,9 +8,13 @@
     <InternalAspNetCoreSdkPackageVersion>2.1.7-build-20210413.1</InternalAspNetCoreSdkPackageVersion>
     <MicrosoftNETCoreAppPackageVersion>2.1.27</MicrosoftNETCoreAppPackageVersion>
 
+    <MicrosoftExtensionsCachingAbstractionsPackageVersion>8.0.0</MicrosoftExtensionsCachingAbstractionsPackageVersion>
+    <MicrosoftExtensionsOptionsPackageVersion>8.0.2</MicrosoftExtensionsOptionsPackageVersion>
+    <MicrosoftExtensionsDependencyInjectionPackageVersion>8.0.1</MicrosoftExtensionsDependencyInjectionPackageVersion>
```

Comment:
Why only these three?  I see many others in this repo that have an LTS version supporting .NETFramework.  Shouldn't you make all of those PackageReferences and delete the code from here?

Reply:
Those are the only ones used by the packages that are still in support - going forward I want to clean up all of the unused code in this repo & dotnet/aspnetcore, but there's quite a bit of it (though admittedly much less in this repo). For now I'm doing the minimal thing to produce a shippable build